.gitlab-ci.yml

```diff
@@ -34,41 +34,41 @@ stages:
   - test
   - publish
 
-build:prepare:raspbian:
+build:prepare:raspios-bullseye:
```

Comment:
Do we even need to name it bullseye if it's the only image which is built here?

Reply:
For the actual CI job you are right, we could drop the `bullseye`.

Thank you. I think it is a good idea so that the next update only needs to deal with `RASPIOS_VERSION` and `CONTAINER_TAG` :+1: